src/main/java/be/syntra/devshop/DevshopBack/models/ProductPageAndMinMaxPrice.java

```diff
@@ -0,0 +1,20 @@
+package be.syntra.devshop.DevshopBack.models;
+
+import be.syntra.devshop.DevshopBack.entities.Product;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.domain.Page;
+
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@Builder
+public class ProductPageAndMinMaxPrice {
```

Comment:
If you add the min-max price to a ProductList without renaming the class, might I suggest we abbreviate this one? :) ProductPage? 
also, fields can be private.

Reply:
renamed to ProductPage
privatized properties <-not a good new habit for me to start....